### PR TITLE
 Partial TL translations for white-paper.md - 1,176 words 

### PR DESCRIPTION
**Pull request by josteinfrancois**
Good day everyone! I have partially translated the mentioned file. Please help me review/improve my work. Thank you.

### CODE REVIEW

File: tl-tl/sc/white-paper.md [hunk 1]
```diff
@@ -2,67 +2,67 @@
 
 ## 1. Paunang Salita
 
-Ang Smart na mga kontrata ay sumasangguni sa anumang kompyuter na program na maaaring awtomatikong magsagawa ng mga termino sa kontrata na naka-preprogram nito. Ang ideya ng 
-Smart contracts refer to any computer program which can automatically execute the terms of its preprogrammed contract. The idea of smart contract was first proposed by the cryptographer Nick Szabo in 1994, making it as old as the Internet itself. Due to the lack of a reliable execution environment, smart contracts have not been widely used.
+Ang matalinong na mga kontrata ay sumasangguni sa anumang kompyuter na program na maaaring awtomatikong magsagawa ng mga termino sa kontrata na naka-preprogram nito. Ang ideya ng matalinong kontrata ay unang iminungkahi ng kriptograper na si Nick Szabo noong 1994, ginagawa itong kasing luma sa Internet. Dahil sa kakulangan ng isang maaasahang environment ng pagpapatupad, ang matalinong mga kontrata ay hindi malawakang nagamit.
```
Comment (burdagay) [inline tl-tl/sc/white-paper.md]: `matalinong na mga kontrata` to `mga matalinong kontrata`

File: tl-tl/sc/white-paper.md [hunk 2]
```diff
@@ -2,67 +2,67 @@
...
+Ang artikulong ito ay magpapakilala sa mga tampok ng NeoContract at magsisiyasat sa hindi teknikal na mga detalye. Mangyaring sumangguni sa teknikal na dokumentasyon para sa teknikal na mga detalye: [docs.neo.org](http://docs.neo.org).
 
-## 2. Features
+## 2. Mga Tampok
 
-### 2.1 Certainty
+### 2.1 Katiyakan
 
-If a program is running on different computers, or at different times on the same computer, the behavior of the program is deterministic if the same input is guaranteed to produce the same output, and vice versa.
+Kung ang isang program ay tumatakbo sa magkaibang mga kompyuter, o sa magkaibang panahon sa parehong kompyuter, ang pag-uugali ng program ay matutukoy kung ang parehong input ay garantisadong makakagawa ng parehong output, at kabaligtaran.
```
Comment (burdagay) [inline tl-tl/sc/white-paper.md]: `pag-uugali` to `pagkilos` since behavior here is a form of action

File: tl-tl/sc/white-paper.md [hunk 3]
```diff
@@ -2,67 +2,67 @@
...
+Ang Blockchain ay isang multi-party na imbakan, at computational na pamamaraan, kung saan ang datos sa loob nitong ipinamamahaging sistema ay ang resulta ng maasahang mga kalkulasyon, at hindi maaaring mabago. Ang matalinong mga kontrata ay gumagana sa loob ng multi-node, ang ipinamamahaging blockchain na network. Kung ang isang matalinong kontrata ay hindi matutukoy, ang mga resulta ng magkaibang mga node ay maaaring hindi pantay-pantay. Bilang resulta, ang pagkasunduan sa pagitan ng mga node ay hindi maaaring maabot, at ang network ay magiging hindi umuunlad. Samakatuwid, sa disenyo ng isang matalinong kontrata na sistema, mayroong isang pangangailangan na mag-alis ng anumang mga kadahilanan na maaaring magsanhi ng hindi matutukoy na pag-uugali.
```
Comment (burdagay) [inline tl-tl/sc/white-paper.md]: `computational ` to `kalkulasyon`

Comment (burdagay) [reply tl-tl/sc/white-paper.md]: `pag-uugali` to `pagkilos`

File: tl-tl/sc/white-paper.md [hunk 4]
```diff
@@ -2,67 +2,67 @@
...
+Ang pagkakaroon ng system-time ay isang sobrang karaniwang function ng sistema, na maaaring mabigat na nailalapat sa tiyak na mga pamamaraan ng time-sensitive na kontrata. Gayunpaman, ang pagkakaroon ng system-time ay isang hindi matutukoy na function ng sistema, at mahirap kumuha ng isang pinag-isa, tumpak na oras sa isang ipinamamahaging na sistema, batay sa mga resulta ng magkaibang mga node ay hindi magkakapantay. Ang NeoContract ay nagbibigay ng isang block-based na system-call na tinatrato ang buong blockchain, bilang isang timestamp na server, at kumukuha ng timestamp kailanman nabuo ang isang bagong bloke. Sa pamantayan, ang NEO na network ay lumilikha ng isang bagong bloke sa bawat 15 na mga segundo, kaya ang kontrata ay tumatakbo sa humigit-kumulang na parehong oras sa pinakabagong block-time, dagdag bawas ng 15 na mga segundo.
```
Comment (burdagay) [inline tl-tl/sc/white-paper.md]: `sobrang karaniwang` to `napakakaraniwang`

Comment (burdagay) [reply tl-tl/sc/white-paper.md]: `function` to `punsyon`

File: tl-tl/sc/white-paper.md [hunk 5]
```diff
@@ -2,67 +2,67 @@
...
...kal na hindi natutukoy na function, at bawat system-call ay kumukuha ng magkaibang mga resulta. Sa isang ipinamamahaging sistema, may maraming mga paraan upang malutas ang problemang ito: Una, ang parehong sapalaran na seed ay maaaring gamiting para sa lahat ng mga node, upang ang return na pagkakasunod-sunod ng buong sapalaran na function ay matutukoy, ngunit ang paraang ito ay naglalantad sa buong sapalaran na resulta nang maaga, labis na nagbabawas ng praktikal na halaga ng sapalarang numero. Isa pang posibleng solusyon, ay hayaan ang lahat ng node na makipag-usap sa isang kolaboratibong paraan upang bumuo ng sapalarang mga numero. Maaari itong makamit sa paggamit ng cryptographic na mga pamamaraan upang lumikha ng isang patas na sapalarang numero, ngunit ang kawalan ay nakapatong sa napakababaw na pagganap, at sa pangangailangan ng karagdagang pananaw sa komunikasyon. Ang isang sentralisadong tagapagtustos ng sapalarang numero ay maaaring gamitin upang bumuo ng sapalarang mga numero na nangangako ng walang pagbabago at pagganap, ngunit ang sagabal sa ganitong paraan ay halata; Ang mga gumagamit ay kailangang walang kondisyong magtiwala sa sentralisadong tagapagtustos ng numero.
```
Comment (burdagay) [inline tl-tl/sc/white-paper.md]: `function` to `punsyon`

Comment (burdagay) [reply tl-tl/sc/white-paper.md]: `maaaring gamiting para sa` to `maaaring gamitin para sa`

Comment (burdagay) [reply tl-tl/sc/white-paper.md]: I don't think `sapalaran` means random. But if it was accepted by others then its okay.

File: tl-tl/sc/white-paper.md (hunk 4 above)
Comment (burdagay) [reply tl-tl/sc/white-paper.md]: `Sa pamantayan` to `Karaniwan,`

File: tl-tl/sc/white-paper.md [hunk 6]
```diff
@@ -2,67 +2,67 @@
...
+2. Ang kontrata na program ay maaaring gumamit ng hash na halaga sa bloke bilang isang generator ng sapalarang numero, dahil ang block hash na halaga ay may tiyak na likas na pagkasapalaran. Ang paraang ito ay maaaring gamitin upang makakuha ng isang mahinang sapalarang numero
 
 #### 2.1.3 Data Source
 
-If a program obtains data at run-time, it might become a non-deterministic program if the data source provides non-deterministic data. For example, using different search engines to obtain the top 10 search results for a particular keyword, may yield different results, in various sort orders, if different IP addresses are used.
+Kung ang isang program ay nakakuha ng datos sa run-time, ito ay maaaring maging isang hindi matutukoy na program kung ang data source ay nagbibigay ng hindi matutukoy na datos. Halimbawa, ang paggamit ng magkaibang mga search engine upang makakuha ng nangungunang 10 na mga resulta sa paghanap para sa isang partikular na keywork, ay maaaring mag-ani ng magkaibang mga resulta, sa iba't ibang paraan ng pagkakaayos, kung magkaibang mga IP address ang ginamit.
```
Comment (burdagay) [inline tl-tl/sc/white-paper.md]: `hindi matutukoy` I would translate this as is to `non-deterministic`.

File: tl-tl/sc/white-paper.md [hunk 7]
```diff
@@ -2,67 +2,67 @@
...
-For smart contracts, NEO provides two types of deterministic data sources:
+Para sa matalinong mga kontrata, ang NEO ay nagbibigay ng dalawang mga uri ng natutukoy na mga data source:
 
-1. **Blockchain Ledger**
+1. **Blockchain na Ledger**
+	
+   Ang kontrata na pamamaraan ay maaaring mag-access sa lahat ng datos sa buong chain gamit ang interoperable na mga serbisyo, kabilang ang kumpletong mga bloke at mga transaksyon, at para sa bawat patlang nila. Ang datos sa mga bloke ay natutukoy at hindi nagbabago, kaya maaari silang ligtas na ma-access gamit ang matalinong mga kontrata.
 
-   The contract procedure can access all data on the entire chain through interoperable services, including complete blocks and transactions, and each of their fields. The data on the blocks are deterministic and consistent, so they can be securely accessed by smart contracts.
+2. **Imbakan ng Kontrata na Lawak**
```
Comment (burdagay) [inline tl-tl/sc/white-paper.md]: `Imbakan ng Kontrata na Lawak` to `Ispasyo Na Imbakan ng Kontrata`

File: tl-tl/sc/white-paper.md [hunk 8]
```diff
@@ -2,67 +2,67 @@
...
 
-2. **Contract Storage Space**
+   Bawat kontratang nai-deploy sa NEO na network, ay mayroong isang pribadong imbakan na lawak na maaari lamang ma-access gamit ang kontrata. Ang mekanismo sa pagkakasundo ng NEO ay sumisigurado sa hindi pagbabago ng estado ng imbakan, sa bawat node sa network.
```
Comment (burdagay) [inline tl-tl/sc/white-paper.md]: `imbakan na lawak` to `ispasiyo na imbakan`

File: tl-tl/sc/white-paper.md [hunk 9]
```diff
@@ -2,67 +2,67 @@
...
 
-   Each contract deployed on the NEO network, has a private storage area that can only be accessed by the contract itself. NEO consensus mechanism ensures consistency of the storage status, of each node in the network.
+Para sa mga sitwason kung saan ang pag-access sa hindi blockchain na datos ay kinakailangan, ang NEO ay hindi nagbibigay ng isang direktang paraan upang makipag-ugnayan sa mga datos na ito.
```
Comment (burdagay) [inline tl-tl/sc/white-paper.md]: `sitwason` i think this is `sitwasyon`?

File: tl-tl/sc/white-paper.md [hunk 10]
```diff
@@ -2,67 +2,67 @@
...
-
-For smart contracts, the execution environment is often more important than the speed of execution of the instruction. Smart contracts are more involved in IO operation of the logic, to determine the instructions, where the implementation of these instructions can easily be optimized. Every time the smart contract is called, it must start up a new virtual machine / container. Therefore, the execution speed of the environment itself (starting a virtual machine / container) has greater impact on the performance of the smart contract system. 
+Para sa matalinong mga kontrata, ang environment ng pagpapatupad ay kadalasang mas importante kaysa sa bilis ng pagpapatupad ng tagubilin. Ang matalinong mga kontrata ay mas kasangkot sa IO na operasyon ng lohika, upang matukoy ang mga tagubilin, kung saan ang implementasyon ng mga tagubiling ito ay madaling mapaunlad. Sa bawat pagkakataon na tinawag nag matalinong kontrata, dapat itong magsimula ng isang panibagong virtual machine / container. Samakatuwid, ang bilis ng pagpapatupad sa environment (pagsisimula ng isang virtual machine / container) ay may mas malaking epekto sa pagganap ng matalinong kontrata na sistema.
```
Comment (burdagay) [inline tl-tl/sc/white-paper.md]: `virtual machine` to `bertuwal na makina`

File: tl-tl/sc/white-paper.md [hunk 11]
```diff
@@ -2,67 +2,67 @@
...
 
-Smart contracts in NeoContract can call each other, but not be recursively called. Recursion can be achieved within the contract, but it cannot cross the boundaries of the current contract. In addition, the call relationship between contracts must be static: The target cannot be specified at runtime. This allows the behavior of the program to be fully determined before execution, and its call relationship to be fully defined before it can run. Based on this, multiple contracts can be dynamically partitioned to achieve parallel execution.
+### 2.2 Mataas na Pagganap
 
-### 2.2 High performance
+Ang environment ng pagpapatupad ng isang matalinong kontrata ay naglalaro ng isang mahalagang papel sa pagganap nito. Kapag ating pinag-aralan ang pagganap sa anumang environment ng pagpapatupad, may dalawang pangunahing tagapagpahiwatig na kritikal:
 
-The execution environment of a smart contract plays an integral role in its performance. When we analyze the performance of any execution environment, there are main two indicators which are critical:
+1. Bilis ng pagpapatupad ng tagubilin
```
Comment (burdagay) [inline tl-tl/sc/white-paper.md:62]: `tagubilin` to `instruksiyon`

**Review by burdagay — CHANGES_REQUESTED** [review]
Please check. Thanks!

File: tl-tl/sc/white-paper.md (hunk 6 above)
Comment (josteinfrancois) [reply tl-tl/sc/white-paper.md]: For technicality, I will change it back to `non-deterministic`.

File: tl-tl/sc/white-paper.md (hunk 11 above)
Comment (josteinfrancois) [reply tl-tl/sc/white-paper.md:62]: Will retain this one.

File: tl-tl/sc/white-paper.md (hunk 5 above)
Comment (josteinfrancois) [reply tl-tl/sc/white-paper.md]: Thank you for your input. Will retain that one.

File: tl-tl/sc/white-paper.md (hunk 10 above)
Comment (josteinfrancois) [reply tl-tl/sc/white-paper.md]: Will change it to `birtwal na makina`.

**Review by josteinfrancois — COMMENTED** [review]
Thank you for your review @burdagay. I have updated the file based on some of your comments. :)

**Comment by burdagay** [comment]
Thanks. This is now okay.

**Review by burdagay — APPROVED** [review]
This is now okay. Thanks!

**Pull request merged**